src/api/app/assets/stylesheets/webui2/datatables.scss

```diff
@@ -11,3 +11,18 @@ table.table-fixed {
   width: 100% !important;
   margin-left: 0 !important;
 }
+
+// TODO: Delete when bug is fixed: https://github.com/mkhairi/jquery-datatables/issues/17
```

Comment:
[nitpick] This could be written in a different way:
```
table.dataTable.table-sm {
  min-width: 100%;

 .sorting:before,
 .sorting_asc:before,
 .sorting_desc:before {
    top: initial;
    right: 1em;
    bottom: 0.3em;
  }

  td { white-space: normal !important; }
}